sktime/datatypes/_vectorize.py

```diff
@@ -271,10 +271,14 @@ def _iter_cols(inst, group_name=None):
 
         iter_levels = self._iter_levels(iterate_as)
         is_self_iter = len(iter_levels) == self.X_mi_index.nlevels
+        if len(iter_levels) == 1:  #
```

Comment:
```suggestion
        if len(iter_levels) == 1:  # single level, groupby expects scalar
```

Assuming you intended to use same comment from below.

Reply:
no, this is an accidental duplication